flycheck-eglot.el

```diff
@@ -218,7 +218,9 @@ ORIG is the original function, (BEG END) is the range"
                                        end))
                               (beg (= beg (flymake-diagnostic-beg s)))
                               (t t)))
-                      eglot--diagnostics)))
+                      (if (consp eglot--diagnostics)
```

Comment:
I'd also add a comment here explaining the change which requires this. (something similar to the PR's description)

Reply:
Added one line of comment.